traffic_portal/app/src/common/modules/widget/routing/widget.routing.tpl.html

```diff
@@ -36,6 +36,20 @@ <h2>Routing Methods</h2>
         </div>
         <div class="clearfix"></div>
     </div>
+    <div class="widget_summary">
+        <div class="w_left w_25">
+            <span>Deep Coverage Zone</span>
+        </div>
+        <div class="w_center w_55">
+            <div class="progress">
+                <div class="progress-bar" role="progressbar" ng-style="{'width': deepCoverageZone + '%'}" style="background-color: #3063BD;"></div>
```

Comment:
Why not use [the `<meter>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter)?
It'd look like:
```html
<div class="widget_summary">
	<div class="w_left w_25">
		<label for="deepCZ-meter">Deep Coverage Zone</label>
	</div>
	<div class="w_center w_55">
		<meter min=0 max=100 ng-value="deepCoverageZone" id="deepCZ-meter">{{ ::deepCoverageZone | number:3}}%</meter>
	</div>
	<div class="clearfix"></div>
</div>
```

Which has the added benefit of a more appropriate `role` than `progressbar`, and adding proper labeling relationships.

Reply:
> Why not use [the `<meter>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter)?
> It'd look like:
> 
> ```
> <div class="widget_summary">
> 	<div class="w_left w_25">
> 		<label for="deepCZ-meter">Deep Coverage Zone</label>
> 	</div>
> 	<div class="w_center w_55">
> 		<meter min=0 max=100 ng-value="deepCoverageZone" id="deepCZ-meter">{{ ::deepCoverageZone | number:3}}%</meter>
> 	</div>
> 	<div class="clearfix"></div>
> </div>
> ```
> 
> Which has the added benefit of a more appropriate `role` than `progressbar`, and adding proper labeling relationships.

I gave that a shot. didn't really work so just going to stick with the current approach.

Reply:
ok, it worked this time with a height and width on it